services/common/src/components/forms/RenderFileUpload.tsx

```diff
@@ -197,9 +196,8 @@ export const FileUpload: FC<FileUploadProps> = ({
         )
       ) {
         notification.error({
-          message: `Failed to upload ${file && file.name ? file.name : ""}: ${
-            errorMessage ? errorMessage : err
-          }`,
+          message: `Failed to upload ${file?.name ?? ""}: ${errorMessage ?? err
```

Comment:
Super minor, but if the file.name is null would this not render as "Failed to upload : {errorMessage}". Noting the presence and placement of the colon.